frontend/packages/client/src/polyfill.js

```diff
@@ -0,0 +1,3 @@
+import { Buffer } from "buffer";
+
+window.Buffer = Buffer;
```

Comment:
this is necessary to fix an error with Buffer that is not available: https://stackoverflow.com/questions/68707553/uncaught-referenceerror-buffer-is-not-defined